### PR TITLE
Lsn1 content - Introduction and setup

### PR DESCRIPTION
**Pull request by CarlyLanglois**
still needs some Windows notes for visual studio setup.
and potentially a sample of importing a project into vs from github.
as well as maybe more material on the project structure?

### CODE REVIEW

File: src/chapters/introduction-and-setup/why-csharp.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/why-csharp.rst]: Syntax highlighting on "for"

File: src/chapters/introduction-and-setup/why-csharp.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/why-csharp.rst]: The way this sentence ends seems a little awkward. I would replace the commas with parentheses to move "another concept we'll cover" more to the side.

File: src/chapters/introduction-and-setup/setup.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/setup.rst]: Maybe rephrase to "We can write and edit code *and* make use of additional features that enhance our coding experience."

File: src/chapters/introduction-and-setup/setup.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/setup.rst]: "... editor, Visual Studio Code."

File: src/chapters/introduction-and-setup/setup.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/setup.rst]: Maybe rephrase to "open-source and available on several operating systems." The "for development" seems unnecessary.

File: src/chapters/introduction-and-setup/setup.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/setup.rst]: Maybe rephrase to "the runtime environment and the virtual machine..."

File: src/chapters/introduction-and-setup/setup.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/setup.rst]: "common programming items, such as date object formatting."

File: src/chapters/introduction-and-setup/setup.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/setup.rst]: This walkthrough needs a little intro. It felt a little out of left field to me.

File: src/chapters/introduction-and-setup/setup.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/setup.rst]: Instead of "related, yet different", what about "an equivalent" or "a different version of Visual Studio called"?

File: src/chapters/introduction-and-setup/creating-a-csharp-project.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/creating-a-csharp-project.rst]: Double quotation mark

File: src/chapters/introduction-and-setup/creating-a-csharp-project.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/creating-a-csharp-project.rst]: "Along with the ``Program.cs`` file"

**Review by gildedgardenia — CHANGES_REQUESTED** [review]
Looks pretty good! I noted somethings that were more personal preference than anything. I didn't know if the Windows instructions had been verified on a Windows machine, so if they have, let me know and I can promptly re-review!

File: src/chapters/introduction-and-setup/why-csharp.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/why-csharp.rst]: This syntax highlighting is not working. Perhaps, the extra "z" is breaking it?

File: src/chapters/introduction-and-setup/why-csharp.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/why-csharp.rst:6]: I would make ".NET framework" bold.

File: src/chapters/introduction-and-setup/windows-setup.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/windows-setup.rst]: This sentence reads a little awkwardly. Perhaps, "Users can select from multiple options for software downloads, such as Visual Studio Code and Visual Studio for Mac, on this page. Do NOT select either of these options!"

File: src/chapters/introduction-and-setup/creating-a-csharp-project.rst
Comment (gildedgardenia) [inline src/chapters/introduction-and-setup/creating-a-csharp-project.rst]: I would remove "the language"

**Review by gildedgardenia — APPROVED** [review]
I like the setup instructions and only noticed a few minor things. Before you merge, you should review your alt text. I think some of it could be more descriptive since certain defaults are selected or arrows are present pointing to key buttons. Maybe more like "Visual Studio for Mac Installation window with aforementioned menu defaults selected."

File: src/chapters/introduction-and-setup/why-csharp.rst
Comment (CarlyLanglois) [reply src/chapters/introduction-and-setup/why-csharp.rst:6]: this is funny, Jim tells me not to bolden anything unless its getting defined so when i wrote this bit for the java book, he had me remove the entry since we dont define the term until later. I've been trying to follow that rule since

**Pull request merged**
